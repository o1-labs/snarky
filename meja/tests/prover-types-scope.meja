type t;

Prover {
  type t1;

  type t2;
};

let f = fun (f : t -> t1 -> t2, x, y) => {
  f(x, y);
};
