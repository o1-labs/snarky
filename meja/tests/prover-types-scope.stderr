File "tests/prover-types-scope.meja", line 9, characters 22-24:
Error: Unbound type constructor t1.
